libs/category/state/src/reducers/category-entities/category-entities.reducer.ts

```diff
@@ -16,6 +16,9 @@ import {
 } from '../../actions/category.actions';
 import { daffCategoryEntitiesAdapter } from './category-entities-adapter';
 
+/**
+ * Returns the state for category entities.
```

Comment:
Say something about the reducer storing category entities in response to certain actions.

Reply:
done